site/soc/software/charts/osh/openstack-tempest/tempest.yaml

```diff
@@ -0,0 +1,100 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: openstack-tempest-soc
+  layeringDefinition:
+    abstract: false
+    layer: site
+    parentSelector:
+      name: tempest-global
+      component: tempest
+    actions:
+      - method: merge
+        path: .
+  storagePolicy: cleartext
+data:
+  wait:
+    timeout: 3000
+{% if deploy_tempest is defined and deploy_tempest is sameas true %}
+  values:
+    # Adding images overrides here instead of versions.yaml
+    images:
+      tags:
+        tempest_run_tests: "{{ suse_osh_registry_location }}/openstackhelm/tempest:{{ suse_infra_image_version }}"
```

Comment:
the pattern is to define the image versions in the versions.yaml and use substitution to override the default value in the global chart.

Reply:
Done.